packages/mdc-drawer/README.md

```diff
@@ -403,7 +403,7 @@ Temporary drawers can easily be initialized using their default constructors as
 ```javascript
 import {MDCTemporaryDrawer} from '@material/drawer';
 
-const drawer = new MDCTemporaryDrawer(document.querySelector('.mdc-temporary-drawer'));
+const drawer = new MDCTemporaryDrawer(document.querySelector('.mdc-drawer--temporary'));
```

Comment:
doesnt look like this should be in this PR

Reply:
This should have been updated in #1730 but we missed it :-\